src/components/Content/subpages/Surveys/Surveys.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import MyTable from "../../../reusable/MyTable";
+import moment from "moment";
```

Comment:
Tip: Check the [date-fns](https://date-fns.org/) library, it has the same capability like `moment` but has only 6.5kB and better functional API without mutations.

Reply:
DONE & thanks for suggesting of the library